public/app/plugins/panel/debug/panelcfg.gen.ts

```diff
@@ -0,0 +1,36 @@
+// Code generated - EDITING IS FUTILE. DO NOT EDIT.
+//
+// Generated by:
+//     public/app/plugins/gen.go
+// Using jennies:
+//     TSTypesJenny
+//     PluginTSTypesJenny
+//
+// Run 'make gen-cue' from repository root to regenerate.
+
+export const PanelCfgModelVersion = Object.freeze([0, 0]);
+
+export type UpdateConfig = {
+  render: boolean,
+  dataChanged: boolean,
+  schemaChanged: boolean,
+};
+
+export type UpdateCounters = {
+  render: number,
+  dataChanged: number,
+  schemaChanged: number,
+};
+
+export enum DebugMode {
+  Cursor = 'cursor',
+  Events = 'events',
+  Render = 'render',
+  State = 'State',
+  ThrowError = 'ThrowError',
+}
+
+export interface DebugPanelOptions {
```

Comment:
This has to be named `PanelOptions`, which is something we need to enforce in pfs. small bug in the way of letting us do that right now, but will have it quite soon